views/marts/product/fct_board_activity.view.lkml

```diff
@@ -0,0 +1,177 @@
+  }
+
+  measure: total_server_daily_active_users {
+    type: sum
+    sql: ${server_daily_active_users} ;;
+    label: "Total DAU (Server)"
+    view_label: "  * Metrics. Active Users"
+  }
+
+  ###
+  ### WAU
+  ###
+
+  dimension: weekly_active_users {
+    type: number
+    sql: ${TABLE}.weekly_active_users ;;
+    label: "WAU (Client)"
+    description: "Number of unique user ids reported for the past 7 days"
+  }
+
+  measure: total_weekly_active_users {
+    type: sum
+    sql: ${weekly_active_users} ;;
+    label: "Total WAU (Client)"
+    view_label: "  * Metrics. Active Users"
+  }
+
+  dimension: server_weekly_active_users {
+    type: number
+    sql: ${TABLE}.server_weekly_active_users ;;
+    label: "WAU (Server)"
+    description: "WAU reported by Boards server activity. Defined as the number of unique users that performed an action in the 24 hours previous to sending the telemetry data."
+  }
+
+  measure: total_server_weekly_active_users {
+    type: sum
+    sql: ${server_weekly_active_users} ;;
+    label: "Total WAU (Server)"
+    view_label: "  * Metrics. Active Users"
+  }
+
+  ###
+  ### MAU
+  ###
+
+  dimension: monthly_active_users {
+    type: number
+    sql: ${TABLE}.monthly_active_users ;;
+    label: "MAU (Client)"
+    description: "Number of unique users for the past 30 days, reported by client telemetry. Equals to the number of unique users that performed an action in the given date and previous 29 days."
+  }
+
+  measure: total_monthly_active_users {
+    type: sum
+    sql: ${monthly_active_users} ;;
+    label: "Total MAU (Client)"
+    view_label: "  * Metrics. Active Users"
+  }
+
+  dimension: server_monthly_active_users {
+    type: number
+    sql: ${TABLE}.server_monthly_active_users ;;
+    label: "MAU (Server)"
+    description: "MAU reported by Boards server activity. Defined as the number of unique users that performed an action in the 30 days previous to sending the telemetry data."
+  }
+
+  measure: total_server_monthly_active_users {
+    type: sum
+    sql: ${server_monthly_active_users} ;;
+    label: "Total MAU (Server)"
+    view_label: "  * Metrics. Active Users"
+  }
+
+
+  ###
+  ### Other server metrics
+  ###
+
+  dimension: count_registered_users {
+    type: number
+    sql: ${TABLE}.count_registered_users ;;
+    label: "Registered users"
+    description: "Total number of registered users. Includes deleted users."
+  }
+
+  measure: total_count_registered_users {
+    type: sum
+    sql: ${count_registered_users} ;;
+    label: "Total Registered Users"
+    view_label: " * Metrics. Registered Users"
+  }
+
+  ###
+  ### Metadata
+  ###
+
+
+  dimension: has_client_data {
+    type: yesno
+    sql: ${TABLE}.has_client_data ;;
+    label: "Has client data?"
+    description: "Whether telemetry data were reported from any client at the current date."
```

Comment:
maybe make it clear that these are Board users (like for servers data)?

Reply:
Updated. Let me know if the new wording looks better.